doc/source/conduce-entities.rst

```diff
@@ -0,0 +1,159 @@
+     - $102,325.26
+     - transit
+   * - 5
+     - freight
+     - 2017-10-24T10:22:18+00:00
+     - 25.029996
+     - -134.318848
+     - $1,431,873,345.01
+     - transit
+
+The data in this table represents the present location of 5 shipments.  Each shipment has an ID, a shipping method, a date when it was last updated and a location in geographic coordinates.  Additionally the current state and value of the shipment is documented.  A Conduce entity is able to capture and represent all of this information.  These data records will be ingested as the first samples of 5 unique entities of the ``shipments`` dataset.
+
+An entity sample requires four fields:
+ **id**
+     a unique string that identifies the entity described by this sample.
+ **kind**
+     the type or category of the object.
+ **time**
+     A :py:class:`datetime.datetime` at which the entity state described by this sample is valid.
+
+And one of:
+ **point**
+     A coordinate pair that defines a location in a 2D cartisian coordinate system.
+ **path**
+     A list of coordinates that define a series of connected line segments in a 2D cartisian coordinate system.
+ **polygon**
+     A list of coordinates that define a closed shape in a 2D cartisian coordinate system.
+
+Each can be specified in (x,y) or (latitude,longitude).  If coordinate types are mixed or multiple coordinate types are specified, a :py:func:`KeyError` will be raised.  If more than one location type is defined, a :py:func:`KeyError` will be raised.
+
+Optionally, a Conduce entity may contain an arbitrary list of attributes.  These may be strings, integers, or floating point numbers.
+
+A source data record maps to a sample as follows::
+
+    {
+        "id": <ID>,
+        "kind": <Method>,
+        "time": <Date> (converted to a datetime object)
+        "point": {
+            "lat": <Longitude>,
+            "lon": <Latitude>
+        },
+        "Value": <Value>,
+        "Status": <Status>
+    }
+
+Example (first record in table)::
+
+    {
+        "id": 1,
+        "kind": "ground",
+        "time": dateutil.parser.parse('2017-10-24T10:22:09+00:00'),
+        "point": {
+            "lon": -91.571045,
+            "lat": 38.022131
+        },
+        "Value": 102325.26,
+        "Status": "delivered" 
+    }
+
+In the example above the ISO-8601 date time string is converted to a :py:class:`datetime.datetime` using :py:func:`dateutil.parser.parse`.
+
+More :ref:`example sample definitions <entity-sample-definitions>`
+
+------------------
+Ingesting entities
+------------------
+
+In order to ingest our source data we must first convert each record into an entity sample as described above.  Once converted, the samples are added to a list.  The list may contain samples for multiple entities.  Once we have created our sample list we call :py:func:`api.ingest_entities`::
+
+    ingest_entities(dataset_id, sample_list, host=app.conduce.com, api-key=00000000-0000-0000-0000-000000000000)
```

Comment:
Is this correct?  Wasn't the intent to have `ingest_samples` vs `ingest_entities`?

Reply:
That change will come with the next PR.  `ingest_samples` does not exist at the time of this writing.